src/gc/gc.cpp

```diff
@@ -11220,14 +11222,15 @@ void gc_heap::adjust_limit_clr (uint8_t* start, size_t limit_size,
             dprintf (3, ("filling up hole [%Ix, %Ix[", (size_t)hole, (size_t)hole + size + Align (min_obj_size, align_const)));
             // when we are finishing an allocation from a free list
             // we know that the free area was Align(min_obj_size) larger
-            size_t free_obj_size = size + Align (min_obj_size, align_const);
+			acontext->alloc_bytes -= size;
```

Comment:
I believe spaces are the default for C#, but tabs are the default for C++. No clue why. :confused: